src/ra_log.erl

```diff
@@ -148,15 +156,20 @@ init(#{uid := UId,
     LogId = maps:get(log_id, Conf, UId),
     ResendWindow = maps:get(resend_window, Conf, ?DEFAULT_RESEND_WINDOW_SEC),
     SnapInterval = maps:get(snapshot_interval, Conf, ?SNAPSHOT_INTERVAL),
+    CPInterval = maps:get(checkpoint_interval, Conf, ?CHECKPOINT_INTERVAL),
```

Comment:
I wonder if we should take the opportunity to try to name these something like `min_checkpoint_interval` We'd have to keep `snapshot_interval` in ra_log_init_args() of but it could be in addition to `min_snapshot_interval`

Reply:
Makes sense to me :+1:

I'll make a follow-up PR for adding `min_snapshot_interval` and falling back to `snapshot_interval`